packages/mdl-textfield/mdl-textfield.scss

```diff
@@ -0,0 +1,356 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/variables";
+@import "mdl-theme/variables";
+@import "mdl-theme/mixins";
+
+$mdl-textfield-error-on-light: #d50000;
+$mdl-textfield-error-on-dark: #ff6e6e;
```

Comment:
Doesn't look like it. However, it is in the stickersheet / mocks as the color ;) 

<img width="671" alt="screen shot 2016-10-03 at 10 44 27" src="https://cloud.githubusercontent.com/assets/1185269/19041545/8108b5d0-8956-11e6-82df-093447be83be.png">